### DISCUSSION TITLE
Storage support

### DISCUSSION THREAD

**Discussion by markrieder** [jellyfin/jellyfin#12362] [General] [answered]
This problem is not new, it appeared a few years ago and it seems that it is still not solved. Jellyfin cannot read external disks and the possibility of connecting a database from an external hdd. The problem can only be solved with command. Please add external drive support to Jellyfin for the less experienced.

**Comment by JPVenson** [comment, marked as answer]
I think you are seriously mixing up terms here. Jellyfin can use external Disks just fine if they are mounted to the OS on windows and linux as both media storage and config storage.

**Comment by markrieder** [comment]
Sorry for the wrong question. The operating system is TrueNAS. The external
hdd is zfs and Jellyfin cannot find the path to this drive. TrueNAS runs on
Linux.

Dňa st 31. 7. 2024, 16:49 JPVenson ***@***.***> napísal(a):
